core/env/docker-compose.yml

```diff
@@ -22,5 +22,16 @@ services:
       POSTGRES_PASSWORD:
     volumes:
       - postgres:/var/lib/postgresql/data
+  minio:
+    image: minio/minio:RELEASE.2023-02-22T18-23-45Z.fips
+    ports:
+      - 127.0.0.1:${MINIO_PORT:-0}:9001
```

Comment:
Variable must be declared in workflow configuration

Reply:
added